bundles/org.openhab.binding.heos/src/main/java/org/openhab/binding/heos/internal/handler/HeosPlayerHandler.java

```diff
@@ -179,6 +179,9 @@ public static void propertiesFromPlayer(Map<String, ? super String> prop, Player
         String serialNumber = player.serial;
         if (serialNumber != null) {
             prop.put(Thing.PROPERTY_SERIAL_NUMBER, serialNumber);
+        } else {
+            prop.put(Thing.PROPERTY_SERIAL_NUMBER, ""); // If no serial number is provided, write an empty string to
```

Comment:
I would like to only use the `pid` as a fallback, so if the serialNumber is empty we use the `pid` on this line.

Reply:
So using the PID as a fake serial number?

Reply:
Yes

Reply:
Okay. Good idea. I will change that and push the change.